taipy/rest/package_desc.md

```diff
@@ -0,0 +1,173 @@
+and limitations under the License.
+
+## What is Taipy REST
+
+Taipy is a Python library for creating Business Applications. More information on our
+[website](https://www.taipy.io). Taipy is split into multiple repositories including
+*taipy-core* and *taipy-rest* to let users install the minimum they need.
+
+Taipy Core mostly includes business-oriented
+features. It helps users create and manage business applications and improve analyses
+capability through time, conditions and hypothesis.
+
+Taipy REST is a set of APIs built on top of the
+*taipy-core* library developed by Avaiga. This project is meant to be used as a complement
+for Taipy and its goal is to enable automation through rest APIs of processes built
+on Taipy.
+
+## Installation
+
+The latest stable version of *taipy-rest* is available through *pip*:
+```bash
+pip install taipy-rest
+```
+
+### Development version
+
+You can install the development version of *taipy-rest* with *pip* and *git* via the taipy repository:
+```bash
+pip install git+https://git@github.com/Avaiga/taipy
+```
+
+This command installs the development version of *taipy* package in the Python environment with all
+its dependencies, including the *taipy-rest* package.
+
+If you need the source code for *taipy-rest* on your system so you can see how things are done or
+maybe participate in the improvement of the packages, you can clone the GitHub repository:
+
+```bash
+git clone https://github.com/Avaiga/taipy.git
+```
+
+This creates the 'taipy' directory holding all the package's source code, and the 'taipy-rest'
+source code is in the 'taipy/rest' directory.
+
+## Configuration
+
+Before running, we need to define some variables. Taipy REST APIs depend on pre-configuration of taipy config objects,
+i.e, is mandatory to define all configuration of DataNodes, Tasks, Sequences, etc. The file containing this
+configuration needs to be passed to the application at runtime. The following variable needs to be defined:
+ - TAIPY_SETUP_FILE: the path to the file containing all of taipy object configuration
+
+If using Docker, the folder containing the file needs to be mapped as a volume for it to be accessible to the
+application.
+
+## Running
+
+To run taipy-rest, you need to install the required development packages.
+We recommend using [Pipenv](https://pipenv.pypa.io/en/latest/) to create a virtual environment
+and install the development packages.
+
+```bash
+pip install pipenv
+pipenv install --dev
+```
+
+To run the application you can either run locally with:
+```bash
+flask run
+```
+
+or it can be run inside Docker with:
+```bash
+docker-compose up
+```
+
+You can also run with a Gunicorn or wsgi server.
+
+### Running with Gunicorn
+
+This project provide a simple wsgi entry point to run gunicorn or uwsgi for example.
+
+For gunicorn you only need to run the following commands
+
+```bash
+pip install gunicorn
+
+gunicorn myapi.wsgi:app
```

Comment:
I thought Taipy core had problems with Gunicorn?

Reply:
I don't know. It's still in the rest/README.md